sale_loyalty_auto_refresh/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,19 @@
+You can set this feature on or off in every company. To do so:
+
+#. Go to *Sales > Configuration > Settings*
+#. In the *Pricing* section you'll find the option *Auto refresh promotions*.
+
+The auto-refresh in the backend is triggered over a minimum set of fields changes. If
+you want to extend the list of that fields:
+
+#. Go to *Settings > Technical > Config parameters*
+#. Add or update the key:
+
+   - For `sale.order`: `sale_loyalty_auto_refresh.sale_order_triggers`
+   - For `sale.order.line`: `sale_loyalty_auto_refresh.sale_order_line_triggers`
+#. In every add the fields seperated by commas that you want to add to the recomputation
+   triggers.
+
+⚠️ After configuring or removing a trigger a restart of Odoo is recommended so the
```

Comment:
I don't understand this statement. Either is required or is not. I'd remove it if not necessary because it's confusing.
I left a comment on the compute method.